webclient/src/websocket/events/SessionEvents.ts

```diff
@@ -130,11 +130,8 @@ function serverIdentification(info: ServerIdentificationData) {
       }
       break;
     case WebSocketConnectReason.REGISTER:
-      if (passwordSaltSupported(serverOptions, webClient)) {
-        SessionCommands.requestPasswordSalt();
-      } else {
-        SessionCommands.register();
-      }
+      const passwordSalt = passwordSaltSupported(serverOptions, webClient) ? generateSalt() : undefined;
```

Comment:
A small NIT (I use a Panda 🐼)

Would `null` be more appropriate here than `undefined`?  It _is_ defined, it's just a nil value.

Reply:
Respectable NIT lol